ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -95,7 +95,7 @@ class ManagementActor(managers: ProcessingTypeDataProvider[DeploymentManager],
           processChangeListener.handle(OnDeployActionFailed(process.id, failure))
         case Right(details) =>
           logger.info(s"Finishing ${beingDeployed.get(process.name)} of $process")
-          processChangeListener.handle(OnDeployActionSuccess(process.id, details.version, details.comment, details.deployedAt, details.action))
+          processChangeListener.handle(OnDeployActionSuccess(process.id, details.version, details.deploymentComment.map(_.value), details.deployedAt, details.action))
```

Comment:
`OnDeployActionSuccess` and other actions connected with deployment should also use DeploymentComment => Option[DeploymentComment]

Reply:
thanks, makes sense, changed it
but unfortunately because of that there was need to get access to DeploymentComment in ui-listener, so after talking with arkadius I moved class to ui-listener